packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputElement.tsx

```diff
@@ -179,52 +182,77 @@ export const TransactionReviewOutputElement = forwardRef<
                     {lines.map(line => (
                         <OutputRightLine key={line.id}>
                             <OutputHeadline>
-                                <Truncate>{line.label}</Truncate>
+                                <Truncate>
+                                    {state === 'active' &&
+                                    (line.id === 'address' || line.id === 'regular_legacy')
+                                        ? line.confirmLabel
+                                        : line.label}
+                                </Truncate>
                             </OutputHeadline>
                             <OutputValue>
                                 <TruncateWrapper condition={hasExpansion}>
-                                    <OutputValueWrapper>
-                                        {line.plainValue ? (
-                                            line.value
-                                        ) : (
-                                            <FormattedCryptoAmount
-                                                disableHiddenPlaceholder
-                                                value={line.value}
-                                                symbol={
-                                                    // TX fee is so far always paid in network native coin
-                                                    line.id !== 'fee' && token
-                                                        ? token.symbol
-                                                        : cryptoSymbol
-                                                }
-                                            />
-                                        )}
-                                    </OutputValueWrapper>
+                                    {state === 'active' &&
```

Comment:
nit: putting this condition in const would help readability

Reply:
I meant `line.id === 'address' || line.id === 'regular_legacy'` condition or whole `isActive && (line.id === 'address' || line.id === 'regular_legacy')` condition, but never mind 😆